Felienne/hedy
# Adds Spanish translations for missing UI text

**Description**

I added a few first draft translations for some missing UI strings. Particularly, I wanted to add strings for error messages — my favourite! 😄

In particular, I took every untranslated key from the `ui` category in the `en.yaml` and ensured there was a translation in `es.yaml`:

https://github.com/Felienne/hedy/blob/5345052fe62a01c9f78cab1bda5489cf1557de04/coursedata/texts/en.yaml#L1-L53

**Fix for**

This addresses (but does not fix!)  #1538

**How to test**

 - Select Spanish (es) from the language drop-down
 - Go to http://localhost:8080/hedy/10

You should see the error message in Spanish:

<img width="1063" alt="Screenshot of a Hedy error message in Spanish" src="https://user-images.githubusercontent.com/2294397/145818585-ac0d81e7-9108-40f8-a734-644d5c47a91f.png">


**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!

**NOTE**
I am a [_heritage_ Spanish speaker](https://en.wikipedia.org/wiki/Heritage_language), and do not have training in translation. **Please have a native Spanish speaker proofread these translations!**
